client/src/components/Heroes/HeroesRadarTab.tsx

```diff
@@ -69,24 +105,62 @@ function HeroesRadarTab({ setLoading, courses }: { setLoading: (arg: boolean) =>
     }
   };
 
+  const exportToCsv = () => {
+    const data = onlyDefined(formData);
+    const formParams = Object.entries(data).reduce((acc: string[][], [key, value]) => {
+      if (key === 'dates' && Array.isArray(value)) {
+        const [startDate, endDate] = value.map(date => date?.format('YYYY-MM-DD'));
+        return [...acc, ['startDate', `${startDate}`], ['endDate', `${endDate}`]];
+      }
+      return [...acc, [key, `${value}`]];
+    }, []);
+
+    const params = new URLSearchParams([['current', '1'], ['pageSize', `${heroes.pagination.total}`], ...formParams]);
+    window.location.href = `/api/v2/gratitudes/heroes/radar/csv?${params}`;
+  };
+
   return (
     <>
-      <Form layout={formLayout} form={form} onFinish={handleSubmit} style={{ marginBottom: 24 }}>
-        <Form.Item name={'courseId'} label="Courses" style={{ minWidth: 260, marginBottom: 16 }}>
-          <Select options={courses.map(({ id, name }) => ({ value: id, label: name }))} />
-        </Form.Item>
-        <Form.Item name={'notActivist'} valuePropName="checked" style={{ marginBottom: 16 }}>
-          <Checkbox>Show only not activists</Checkbox>
-        </Form.Item>
-        <Space align="start" size={20}>
-          <Button size="middle" type="primary" htmlType="submit">
-            Filter
-          </Button>
-          <Button size="middle" type="primary" onClick={onClear}>
-            Clear
+      <Row style={{ marginBottom: 24 }} justify="space-between">
+        <Form layout={formLayout} form={form} onFinish={handleSubmit}>
+          <Form.Item name={'courseId'} label="Courses" style={{ minWidth: 260, marginBottom: 16 }}>
+            <Select
+              placeholder="Select course"
+              showSearch
+              optionFilterProp="label"
+              options={courses.map(({ id, name }) => ({ value: id, label: name }))}
+            />
+          </Form.Item>
+          {isAdmin && (
+            <Form.Item name={'countryName'} label="Countries" style={{ minWidth: 260, marginBottom: 16 }}>
+              <Select
+                placeholder="Select country"
+                showSearch
+                options={countries.map(({ countryName }) => ({ value: countryName, label: countryName }))}
+              />
+            </Form.Item>
+          )}
+          <Form.Item name={'dates'} label="Dates" style={{ minWidth: 260, marginBottom: 16 }}>
+            <RangePicker presets={rangePresets} />
+          </Form.Item>
+          <Form.Item name={'notActivist'} valuePropName="checked" style={{ marginBottom: 16 }}>
+            <Checkbox>Show only not activists</Checkbox>
+          </Form.Item>
+          <Space align="start" size={20} style={{ marginBottom: 16 }}>
+            <Button size="middle" type="primary" htmlType="submit">
```

Comment:
middle is default value

Reply:
got it, I've copy-pasted it from Gratitudes form, should I fix it there too?

Reply:
fixed in `HeroesRadarTab.tsx`